### DISCUSSION TITLE
Key error when opening example NWB file

### DISCUSSION THREAD

**Discussion by twheatcroft07** [dandi/helpdesk#141] [Q&A] [answered]
I downloaded an example nwb file ([from here](https://dandiarchive.org/dandiset/000363/0.230822.0128/files?location=sub-440956&page=1)). I want to open it and do some simple analysis (e.g. find the proportion of units that kilosort labelled 'good').

I tried to use code from the ecephys example jupyter notebook. I tried to run the code in the below screenshot, but it gives me an error. I know the dataset contains ecephys, and I can tell from the nwbwidget that the data I've downloaded has the appropriate info.

I'm sure this is a beginner's mistake, but I'm not sure how to get past it. Can anyone help?

![image](https://github.com/dandi/helpdesk/assets/116670057/211470d5-2911-47d7-9d87-470a6c5f56bf)

**Comment by satra** [comment, marked as answer]
this particular file does not have anything under `processing`. on the file explorer, you can open the file in your browser using Neurosift. Click on open with and choose neurosift.

here is a direct link: https://neurosift.app/?p=/nwb&url=https://api.dandiarchive.org/api/assets/e1e626c9-f11e-4316-95b3-44c4f485d3ef/download/&dandisetId=000363&dandisetVersion=0.230822.0128

**Reply by twheatcroft07** [reply]
Thankyou!
